includes/wikia/models/DesignSystemGlobalFooterModel.class.php

```diff
@@ -159,12 +159,12 @@ public function getData() {
 					'type' => 'line-text',
 					'title' => [
 						'type' => 'translatable-text',
-						'key' => 'global-footer-community-apps-header'
+						'key' => 'global-footer-fandom-app-header'
```

Comment:
What about the case that `The FANDOM App` string is only for english?

Reply:
I'm aware of that, this was submitted before we agreed to change the ACs